File: sBuilding/Trunk/Client/sBuilding/sTimeKeeping/Model/EventImportObject.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;

namespace sTimeKeeping.Model
{
    [DataContract]
    public class EventImportObject
    {
         [DataMember]
        public long OrgId { get; set; }
         [DataMember]
        public long SubOrgId { get; set; }
         [DataMember]
        public string EventName { get; set; }
         [DataMember]
        public string HourBegin { get; set; }
         [DataMember]
        public string Date { get; set; }
         [DataMember]
        public int HourKeeping { get; set; }
         [DataMember]
        public string Description { get; set; }
         [DataMember]
        public string MemberCode { get; set; }
         [DataMember]
        public string MemberName { get; set; }
    }
}

File: sBuilding/Trunk/Server/Java/timekeepingservice/src/com/swt/timekeeping/customer/object/EventImportObject.java
package com.swt.timekeeping.customer.object;

import java.io.Serializable;
/**
 * @author Trang-PC
 */
public class EventImportObject implements Serializable{
	/**
	 */
	private static final long serialVersionUID = 1L;
	private long OrgId;
	private long SubOrgId;
	private String EventName;
	private String HourBegin;
	private String Date;
	private int HourKeeping;
	private String Description;
	private String MemberCode;
	private String MemberName;
	
	public long getOrgId() {
		return OrgId;
	}
	public void setOrgId(long orgId) {
		OrgId = orgId;
	}
	public long getSubOrgId() {
		return SubOrgId;
	}
	public void setSubOrgId(long subOrgId) {
		SubOrgId = subOrgId;
	}
	public String getEventName() {
		return EventName;
	}
	public void setEventName(String eventName) {
		EventName = eventName;
	}
	public String getHourBegin() {
		return HourBegin;
	}
	public void setHourBegin(String hourBegin) {
		HourBegin = hourBegin;
	}
	public String getDate() {
		return Date;
	}
	public void setDate(String date) {
		Date = date;
	}
	public int getHourKeeping() {
		return HourKeeping;
	}
	public void setHourKeeping(int hourKeeping) {
		HourKeeping = hourKeeping;
	}
	public String getDescription() {
		return Description;
	}
	public void setDescription(String description) {
		Description = description;
	}
	public String getMemberCode() {
		return MemberCode;
	}
	public void setMemberCode(String memberCode) {
		MemberCode = memberCode;
	}
	public String getMemberName() {
		return MemberName;
	}
	public void setMemberName(String memberName) {
		MemberName = memberName;
	}

}
